clash-lib/src/Clash/Util/Interpolate.hs

```diff
@@ -0,0 +1,363 @@
+{-|
+Copyright  :  (C) 2019, QBayLogic B.V.
+                  2013, Nikita Volkov
+License    :  BSD2 (see the file LICENSE)
+Maintainer :  Christiaan Baaij <christiaan.baaij@gmail.com>
+-}
+
+{-
+This is an adaptation of
```

Comment:
How much effort is to to highlight what's new?

Reply:
No effort at all! Original code is almost entirely left untouched, I just added some post-processing.

Reply:
Done